main.js

```diff
@@ -0,0 +1,117 @@
+require('dotenv').config();
+const visitManagerUtils = require('./src/visitManagerUtils');
+const authUtils = require('./src/authUtils');
+
+function parse_args() {
```

Comment:
Solo para mantener convencion:
```suggestion
function parseArgs() {
```